src/app/core/app.component.html

```diff
@@ -1,10 +1,13 @@
 <div class="container" *ngrxLet="session$ as session">
-  <div class="left" [ngClass]="{ collapsed: !leftMenuAndHeaders, expanded: leftMenuAndHeaders }">
+  <div class="left" *ngrxLet="leftMenuAndHeaders$ as leftMenuAndHeaders" [ngClass]="{ collapsed: !leftMenuAndHeaders, expanded: leftMenuAndHeaders }">
     <alg-left-menu></alg-left-menu>
   </div>
   <ng-container *ngIf="currentMode$ | async as currentMode">
-    <div class="top-bar"
-      [ngClass]="{ 'with-mode-bar': currentMode !== 'normal', collapsed: !leftMenuAndHeaders, expanded: leftMenuAndHeaders }">
+    <div
+      class="top-bar"
+      *ngrxLet="leftMenuAndHeaders$ as leftMenuAndHeaders"
```

Comment:
Just use `leftMenuAndHeaders` defined above in this block, no need for another subscription (`ngrxlet`)

Reply:
They're not actually in the same block, so if I remove it it stops working.